diff --git a/webrtc/base/network.h b/webrtc/base/network.h
index 52d7d35a2..aff145dc3 100644
--- a/webrtc/base/network.h
+++ b/webrtc/base/network.h
@@ -144,7 +144,7 @@ class NetworkManagerBase : public NetworkManager {
   void GetNetworks(NetworkList* networks) const override;
   void GetAnyAddressNetworks(NetworkList* networks) override;
   // Defaults to true.
-  bool ipv6_enabled() const { return ipv6_enabled_; }
+  bool ipv6_enabled() const { return false; }
   void set_ipv6_enabled(bool enabled) { ipv6_enabled_ = enabled; }
 
   void set_max_ipv6_networks(int networks) { max_ipv6_networks_ = networks; }
